ManiVault/src/plugins/TextData/src/OrderedMap.cpp

```diff
@@ -77,7 +77,7 @@ namespace
         loadFromDisk(variantMap, combinedList);
 
         // Iterate over the big stringlist and extract the key-value pairs
-        for (int i = 0; i < combinedList.size(); )
+        for (size_t i = 0; i < static_cast<size_t>(combinedList.size()); )
```

Comment:
As .size() returns a sizetype_t would it not be cleaner to make the type of i sizetype_t?